python/lsst/summit/extras/focusAnalysis.py

```diff
@@ -52,6 +52,10 @@ class FitResult:
     sigma: float
 
 
+def getFocusFromHeader(exp):
+    return float(exp.visitInfo.focusZ)
```

Comment:
Is it correct to refer to the `visitInfo` as a “header”?  Also, this one-liner seems like overkill for its own (undocumented) function 😉

Reply:
Fair cop on both accounts - I've renamed it to `getFocusFromExposure` and added docs. I think a function is good though, because even though it's a one-liner, if/when it moves again, or we find out the we need to apply some weird offset/transform from the EFD, we only have to change that in one place.